"2019-08-20 08:46:30,632 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-20 08:46:30,632 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-20 08:46:30,763 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-20 08:46:30,763 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-20 08:46:43,807 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-20 08:47:02,003 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-20 08:47:18,519 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-20 08:47:35,261 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '2', but this can be ignored sometimes"
"2019-08-20 08:47:52,460 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-20 08:48:09,151 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-20 08:48:43,171 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-20 08:49:00,037 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-20 08:49:16,726 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
